catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -65,7 +64,9 @@
   private static final String DATE_TIME =
       CswConstants.XML_SCHEMA_NAMESPACE_PREFIX.concat(":dateTime");
 
-  private static final int RETRY_INTERVAL = 30;
+  private static final long DEFAULT_RETRY_INTERVAL = TimeUnit.SECONDS.toMillis(30);
```

Comment:
❓ Do these need to be configurable? If so, that should be documented as well.

Reply:
I don't really think so but if you feel like they should be then I would recommend that they be configurable via system property that we don't document.

Reply:
We need to stop exposing configurations left and right. I say keep it hidden.